SPECS/libbpf/0001-libbpf-add-txtime-field-in-xdp_desc-struct.patch

```diff
@@ -0,0 +1,28 @@
+From c23ca903e76b7a6a23b0e984fdd092826f930328 Mon Sep 17 00:00:00 2001
+From: "Mohamad Azman, Syaza Athirah" <syaza.athirah.mohamad.azman@intel.com>
+Date: Thu, 16 Jul 2020 18:00:51 -0400
+Subject: [PATCH] libbpf: add txtime field in xdp_desc struct
+
+Add a xdp_desc entry to specify packet LaunchTime for drivers
+that support the feature.
+
+Signed-off-by: Mohamad Azman, Syaza Athirah <syaza.athirah.mohamad.azman@intel.com>
+---
+ include/uapi/linux/if_xdp.h | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/include/uapi/linux/if_xdp.h b/include/uapi/linux/if_xdp.h
+index a78a809..7c5168c 100644
+--- a/include/uapi/linux/if_xdp.h
++++ b/include/uapi/linux/if_xdp.h
+@@ -104,6 +104,7 @@ struct xdp_desc {
+ 	__u64 addr;
+ 	__u32 len;
+ 	__u32 options;
++	__u64 txtime;
```

Comment:
Could you please make sure we have the corresponding kernel patch?

Reply:
This header aligns with kernel and corresponding change would need to be in kernel as well.

Reply:
Thanks, but we don't use this kernel. Including the link here for future: https://github.com/intel/linux-intel-lts/commit/90e81216a3f9a2714b3f004e7da40f42904ad52b

Reply:
Hi all,
I've check with our EMT3 latest kernel/kernel-rt sources, its including the headers 
include/uapi/linux/if_xdp.h with --> __64 texttime;, line 155.

both kernel/kernel-rt is using the same source tag lts-v6.12.30-emt-250524T131939Z
https://github.com/intel/linux-intel-lts/blob/lts-v6.12.30-emt-250524T131939Z/include/uapi/linux/if_xdp.h#L155